src/main.tsx

```diff
@@ -83,6 +86,21 @@ const useStyles = makeStyles((theme: Theme) =>
 const AppContainer: FC<AppState> = () => {
   const classes = useStyles();
 
+  // redux actions (dont work from import)
+  const signIn = () => {
```

Comment:
Move to separate file please. And you can use `const signIn = () => ({ type: SIGN_IN })`
And create separate constants for action type SIGN_IN, SIGN_OUT